common/templates/errorDialog.modal.html

```diff
@@ -5,7 +5,8 @@ <h3 class="modal-title">Error: {{::ctrl.params.errorCode}}</h3>
 <div class="modal-body">
     {{::ctrl.params.message}}
     <br><br>
-    Click OK to return to the {{::ctrl.params.pageName}}. 
+    Click OK to {{ctrl.params.errorCode == 'Multiple Records Found'? 'redirect ': 'return '}} to the {{::ctrl.params.pageName}}.
```

Comment:
I talked with @hongsudt , she prefers using "go" instead of "return" or "redirect". So just change it to "Click OK to go to the".